sdk/openai/Azure.AI.OpenAI/src/Custom/AzureOpenAIClientOptions.cs

```diff
@@ -1,17 +1,47 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System.ClientModel.Primitives;
 using OpenAI;
 
 namespace Azure.AI.OpenAI;
 
 /// <summary>
 /// Defines the scenario-independent, client-level options for the Azure-specific OpenAI client.
 /// </summary>
-public partial class AzureOpenAIClientOptions : OpenAIClientOptions
+public partial class AzureOpenAIClientOptions : ClientPipelineOptions
 {
-    internal string Version => _version;
-    private readonly string _version;
+    internal string Version { get; }
+
+    /// <summary>
+    /// The authorization audience to use when authenticating with Azure authentication tokens
+    /// </summary>
+    /// <remarks>
+    /// By default, the public Azure cloud will be used to authenticate tokens. Modify this value to authenticate tokens
+    /// with other clouds like Azure Government.
+    /// </remarks>
+    public AzureOpenAIAuthorizationAudience? AuthorizationAudience
```

Comment:
This property should just be called "Audience" to align with the pattern used in the other Azure libraries.